.circleci/db-migration.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+#
+# Copyright 2018-2022 contributors to the Marquez project
+# SPDX-License-Identifier: Apache-2.0
+#
+# Usage: $ ./db-migration.sh
+
+readonly POSTGRES_VERSION="12.1"
+readonly MARQUEZ_VERSION="latest"
```

Comment:
Anybody who builds marquez regularly is going to have this image tag all the time. It'll still run in CI, which is fine, but I wonder whether there's any way to fetch the latest released version so that people can catch mistakes locally rather than waiting for the CI build to fail.

Reply:
Very doable. We running [`new-version.sh`](https://github.com/MarquezProject/marquez/blob/main/new-version.sh), we can also bump `MARQUEZ_VERSION `. Which is better than just `latest` and assuming the version. Also helps with debugging.